### ISSUE TITLE
feature request : a tutorial about multichannel cable

### ISSUE THREAD

**Issue by avilleret** [jamoma/JamomaMax#842]
since this is a great feature of jamoma, I think it should be some words about multichannel cable in Max.
For instance, I'm wondering how to know how many channels  a multichannel cable is carrying ?

**Comment by lossius** [comment]
Good point. For the record, the upper limit currently seems to be 250 channels.
